### ISSUE TITLE
task crashes when run

### ISSUE THREAD

**Issue by tadasajon** [fuelen/ecto_erd#8]
I just installed this library and tried to run ` mix ecto.gen.erd`.  It immediately crashed with this error:
```
** (UndefinedFunctionError) function SpecialUser.__schema__/1 is undefined (module SpecialUser is not available)
    SpecialUser.__schema__(:source)
    (ecto_erd 0.4.0) lib/ecto/erd/graph.ex:134: Ecto.ERD.Graph.from_relation_struct/1
    (elixir 1.13.0-rc.0) lib/enum.ex:4076: Enum.flat_map_list/2
    (ecto_erd 0.4.0) lib/ecto/erd/graph.ex:79: Ecto.ERD.Graph.components/2
    (elixir 1.13.0-rc.0) lib/enum.ex:4076: Enum.flat_map_list/2
    (elixir 1.13.0-rc.0) lib/enum.ex:4077: Enum.flat_map_list/2
    (ecto_erd 0.4.0) lib/ecto/erd/graph.ex:9: Ecto.ERD.Graph.new/2
    (ecto_erd 0.4.0) lib/mix/tasks/ecto.gen.erd.ex:138: Mix.Tasks.Ecto.Gen.Erd.run/
```
I am able to create `SpecialUser`s in my app when I am using it and my app works fine in this regard, so I am not sure why this library is complaining.

**Comment by fuelen** [comment]
@tadasajon have you tried removing `_build`?
The library simply uses all modules which have `__schema__/1` function https://github.com/fuelen/ecto_erd/blob/1dd8b9a4f85dd10f435f3cbf779cd6f95b375b5e/lib/ecto/erd/schema_modules.ex#L4-L16

**Comment by tadasajon** [comment]
Thanks much for responding.

I have a file in my codebase that looks like this:
```
defmodule Quizdrill.Accounts.SpecialUser do
  use Ecto.Schema
  import Ecto.Changeset

  schema "special_users" do
    field :special_id, :integer
    field :first_name, :string
    field :last_name, :string
    field :username, :string

    timestamps()
  end

  @doc false
  def changeset(special_user, attrs) do
    special_user
    |> cast(attrs, [:special_id, :first_name, :last_name, :username])
    |> validate_required([:special_id])
    |> unique_constraint(:special_id)
  end
end
```
I am assume that because I have the line `use Ecto.Schema` that there will be a `__schema__/1` function.  Am I incorrect?

**Comment by fuelen** [comment]
Yes, `use Ecto.Schema` defines `__schema__/1` function.

Check your other modules, which have a line like `belongs_to :special_user, SpecialUser`. Seems like one of those modules references to the module which doesn't exist. Perhaps you don't have declared alias for `Quizdrill.Accounts.SpecialUser` in one of those modules.

**Comment by fuelen** [comment]
Hey @tadasajon 
Any luck?

**Comment by fuelen** [comment]
@jjl Hello
Yes, PR is welcome :)

**Comment by fuelen** [comment]
Resolved in #13
